tests/annotations/tests.py

```diff
@@ -119,17 +119,84 @@ def test_basic_annotation(self):
         for book in books:
             self.assertEqual(book.is_book, 1)
 
+    def test_basic_alias(self):
```

Comment:
I think we should move all new tests to a separate class, e.g. `AliasTests`.

Reply:
@felixxm Does it need a separate module or can be kept in `annotations/tests.py`?

Reply:
We can keep it in the same file.